apps/website/src/app/page.tsx

```diff
@@ -12,10 +13,11 @@ const sora = Sora({
 
 export default function Home() {
     return (
-        <VStack>
-            <VStack h={{ base: "718", sm: "734", md: "724" }} justify="center" spacing="20" position="relative">
+        <>
+            <VStack pt="170px" pb={{ base: "128px", md: "170px" }} justify="center" spacing="20" position="relative">
```

Comment:
Removing the wrapping `<VStack>` tag leads to unwanted change below

![photo_2024-01-12_18-12-07](https://github.com/semaphore-protocol/semaphore/assets/20580910/46c84ed3-118a-42f3-9e2f-7a797e4b602a)